config/routes.rb

```diff
@@ -87,6 +87,7 @@
   get '/import/download_guidance', to: 'import#download_guidance'
 
   get '/patients/:id/household_removeable', to: 'patients#household_removeable'
+  get '/monitoree_unavailable', to: 'patients#monitoree_unavailable'
```

Comment:
Nitpick, but can you just move this to either right before or right after all the `/patients` routes

Reply:
moved here: https://github.com/SaraAlert/SaraAlert/pull/736/commits/24eb9e95f3781d1067cbb86d3926b45d6e3fc1c0